yarn.lock

```diff
@@ -1212,10 +1205,17 @@
     once "^1.4.0"
     universal-user-agent "^4.0.0"
 
-"@octokit/types@^2.0.0", "@octokit/types@^2.0.1", "@octokit/types@^2.11.1":
-  version "2.12.2"
-  resolved "https://registry.yarnpkg.com/@octokit/types/-/types-2.12.2.tgz#e9fbffa294adb54140946d436da9f73bc94b169c"
-  integrity sha512-1GHLI/Jll3j6F0GbYyZPFTcHZMGjAiRfkTEoRUyaVVk2IWbDdwEiClAJvXzfXCDayuGSNCqAUH8lpjZtqW9GDw==
+"@octokit/types@^2.0.0", "@octokit/types@^2.0.1":
```

Comment:
Non-flat. Is this going to be a problem?

Reply:
This could be interesting, but it's not something we can fix, as it's caused by other dependencies. Shouldn't matter anyway though as we don't compile this package using typescript.